include/operations/blas_constants.h

```diff
@@ -265,4 +265,16 @@ struct constant_pair {
 
 }  // namespace blas
 
+#define PORTBLAS_DEVICE_COPY(T1, T2)                             \
+  template <>                                                    \
+  struct sycl::is_device_copyable<blas::IndexValueTuple<T1, T2>> \
+      : std::true_type {};
+
+PORTBLAS_DEVICE_COPY(int32_t, float)
+PORTBLAS_DEVICE_COPY(int32_t, double)
+PORTBLAS_DEVICE_COPY(int64_t, float)
+PORTBLAS_DEVICE_COPY(int64_t, double)
+
+#undef PORTBLAS_DEVICE_COPY
```

Comment:
Thank you for the fix @muhammad-tanvir-1211 
I have just a minor consideration, did you think about using only a template to fix the issue? I think that something like the following should work:

```C++
template<typename ind_t, typename val_t>
struct sycl::is_device_copyable<blas::IndexValueTuple<ind_t, val_t>>
      : std::true_type {};
```

Reply:
Thank you for the suggestion @s-Nick. I have applied the patch in the latest commit.